registrations/models.py

```diff
@@ -71,3 +71,24 @@ class Meta:
 
     def __str__(self):
         return self.domain_name
+
+class UserExternalToken(models.Model):
+    """ External token for user """
+
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        verbose_name=_('user'),
+        on_delete=models.CASCADE,
+    )
+
+    created_at = models.DateTimeField(verbose_name=_('created at'), auto_now_add=True)
+    token = models.TextField(verbose_name=_('token'), editable=False)
+    expire_timestamp = models.DateTimeField(verbose_name=_('expire timestap'), null=True, blank=True)
+    is_disabled = models.BooleanField(verbose_name=_('is disabled?'), default=False)
```

Comment:
For now, let's comment it out and add a note, as we haven't implemented blacklist in this phase.